### PR TITLE
Feature/saygofish

### PR DESCRIPTION
**Pull request by RobertWheel**
says go fish

### CODE REVIEW

File: CardGamesSprint4.py [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+  #How many cards are in the deck assigned to "num_cards"
+  num_cards = len(deck)
+  #For each iteration the number of cars in the deck decrease by 1 card
+  for i in range(num_cards):
+    print("There are "+ str(num_cards - i) +" more cards in the deck")
+#when the deck reaches 0, then the game is over
+  if num_cards == 0:
+      print("The deck is now empty. Game over, Thanks for playing!")
+
+  def sayGoFish():
```
Comment (andredavii) [inline CardGamesSprint4.py:89]: Looks good. Function does the job. I'd say add some comments but it's simple enough to follow along.

Comment (RobertWheel) [reply CardGamesSprint4.py:89]: Sure, I think that's a great idea,

File: source/CardGames.py [hunk 2]
```diff
@@ -12,7 +12,26 @@ def find_root_dir():
     os.chdir('..')
     cwd = os.path.join( cwd, '..')
   return cwd
-
+def startUp():
```
Comment (andredavii) [inline source/CardGames.py:17]: The startup looks good. Instructions are clear. I'd suggest changing the "Go Fish Card Game" title to "Welcome to Go-Fish!". It'll make it more official and exciting for the players.

Comment (RobertWheel) [reply source/CardGames.py:17]: Thanks, I'll definitely do that, it'll make it more welcoming 👍

File: CardGamesSprint4.py [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+    response = input("Do you have any of the requested cards? (YES/NO): ")
+    if response.upper() == "NO":
+        return "GO FISH!!"
+    elif response.upper() == "YES":
+        return ""
+    else:
+        return "Looks like there was a typo in your response, please try again"
+
```
Comment (psk44) [inline CardGamesSprint4.py:97]: Easy to understand and well made! Great job with error handling as well!

File: source/CardGames.py [hunk 4]
```diff
@@ -14,7 +14,26 @@ def find_root_dir():
...
+  print("")
+  print("Instructions:")
+  print("2. Each player is dealt 7 cards at the beginning of the game.")
+  print("3. The remaining cards are placed in a draw pile.")
+  print("4. Players take turns asking each other for cards.")
+  print("5. If a player has the requested card, they must give it to the requesting player.")
+  print("6. If a player does not have the requested card, they must tell the requesting player to 'Go Fish'.")
+  print("7. If the card pile runs out, the game is over.")
+  print("8. The player with the most sets of 4 cards at the end of the game wins.")
+
```
Comment (psk44) [inline source/CardGames.py:34]: Great job Robert! Looks good!

**Pull request merged**